src/stcal/saturation/saturation.py

```diff
@@ -0,0 +1,96 @@
+import logging
+
+import numpy as np
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
```

Comment:
We should not be setting log levels in any library code like this.

Reply:
This is a carryover from the kludge needed in the jwst package, due to the odd way the logging mechanism interacts with stpipe (or something). If you don't have line 6 in there, you won't ever be able to get anything other than INFO level messages out of the function. This is not actually setting the logging level to DEBUG for every usage, it's just *enabling* DEBUG level to be used when desired. 

Of course if someone has fixed all of this nonsense in stcal, such that this is no longer needed, then ignore everything I just said (a lot of people do anyway).

Reply:
There are no debug-level log messages in this code, so I'd suggest we don't need it regardless.  But yeah, we should get that fixed in `stpipe`.